compiler/rustc_infer/src/infer/error_reporting/suggest.rs

```diff
@@ -1,3 +1,4 @@
+use crate::errors::SuggAddLetForLetChains;
```

Comment:
why did you move this?

Reply:
./x fmt did, I guess?

Reply:
I'm surprised if `x fmt` would do this, since otherwise CI would've been broken on tidy before this PR. Anyways, can you revert it? Just a nitpick.

Reply:
of course